packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist_Scoring.res

```diff
@@ -14,3 +14,20 @@ module KLDivergence = {
       quot < 0.0 ? Error(Operation.ComplexNumberError) : Ok(-.answerElement *. logFn(quot))
     }
 }
+
+/*
+
+*/
+module LogScore = {
+  let logFn = Js.Math.log
+  let integrand = (priorElement: float, predictionElement: float, ~answer: float) => {
+    if answer == 0.0 {
+      Ok(0.0)
```

Comment:
?? So if the prediction was normal(0,5), and the answer is 0, the score is 0? Maybe I'm misunderstanding that?

Reply:
I think I misread my notes from my meeting with Nuño. I think it should be `if prediction.pdf(answer)` == 0 then Ok(0)`